Tool feature

nmcayouette opened the pull request.

cayouetten committed:
ReadMe Edit

SSH for proper Repo updated

cayouetten committed:
Feature branch

Minor commenting updates, feature branch creation and commit

LukeBillington commented on FormGenerator.js:
```diff
@@ -1,4 +1,4 @@
-// HTML Form generation tool
+// HTML Form Generation Tool
```
Looking great! Suggest trying to compress the various input types into a function, to reduce code repetition but the level of attributes you provide is great. Also, on the errors, consider using an error object as mentioned on FSO, it's really simple `throw new Error(message)`. Keep it up!

nmcayouette replied:
I updated the attribute addition into a single function to reduce code repetition. Error reporting functionality may be implemented better to come, but am going to leave as simple console for now.

LukeBillington reviewed (changes requested):
- Compress inputs into a function
- Use error objects

cayouetten committed:
Review Updates

Brought all attribute addition into a single method called 'attributes'; and line breaks injected for better result rendering.

LukeBillington commented:
Nice! LGTM
